### ISSUE TITLE
Warhammer 40,000: Space Marine (55150)

### ISSUE THREAD

**Issue by FreeSevenSix** [ValveSoftware/Proton#934]

**Comment by FreeSevenSix** [comment]
installs fine on linux mint 19 cinnamon with nvidia driver 396.54. does not launch, no error message given

system information: https://gist.github.com/FreeSevenSix/96ac1aee67e5825b55cc7245141d4277

**Comment by kisak-valve** [comment]
Hello @FreeSevenSix, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options and drag and drop the generated $HOME/steam-$APPID.log into the comment box.

**Comment by FreeSevenSix** [comment]
i'm sorry,

i did what you asked in the post above, but i am unable to find any file thats called appid.log or any .log file related to the game or its appid

**Comment by efce** [comment]
Hi, 
can confirm that it does not launch.
GF 960M - blob 396.54
I5-6300HQ
8 GB RAM
Manjaro, KDE 5.49
Proton 3.7.5-Beta
[GIST INFO](https://gist.github.com/efce/96f99612651f0ba73cd929c9a62271c3)

[steam-55150.log](https://github.com/ValveSoftware/Proton/files/2338112/steam-55150.log)

**Comment by RCal-Dev** [comment]
Just got the game free via Humble Bundle, can also confirm it doesn't launch in Kubuntu using Kernel 4.18.5.

System info: 
https://gist.github.com/RaptorRCX/32f24bb8df47e80d42dc254dca7556f9

**Comment by ghost** [comment]
Distro: Ubuntu 18.04.1
GPU: Intel Iris Pro 580
Resolution: 1280x720
RAM: 32GB
Controller: DS4
Proton 3.7-5 Beta

https://gist.github.com/daasuharo/cb605b8cd50227982a9bfa4dfcb1f635

[steam-55150.log](https://github.com/ValveSoftware/Proton/files/2338637/steam-55150.log)

**Comment by lineber** [comment]
I tried changing the Windows version to XP and Vista and placing it in a Desktop environment. Couldn't get it to work. Also received it free from Humble Bundle today.

https://gist.github.com/lineber/9a75368b3e6dc1274dae29d98b1d7851

**Comment by alex9099** [comment]
Same, not starting,
Native: https://ptpb.pw/EgW5
Runtime: https://ptpb.pw/S30D

**Comment by alex9099** [comment]
This game apparently seems to have a workarround to play http://www.gamersonlinux.com/forum/threads/warhammer-40k-space-marine-guide.2215/

It needs corefonts, d3dx9,tahoma,vcrun2005,vcrun2008,vcrun2010

**Comment by Xeers** [comment]
Reborn OS (Arch Linux)
Proton 3.7-3/-4beta/-5beta
Nvidia Geforce GTX 950
396.54-1
CPU AMD FX
https://gist.github.com/Xeers/219b4b3e5eb10c6ec225d7cc8c61be44

Pastebin:
https://paste.drhack.net/?74e5bfa4c96021aa#etOLl3BLfiEhkSjBdvPUEMspm26bjb/jlNHVSt6VBIM=

**Comment by CarlB86** [comment]
Linux Mint 19 (ubuntu 18.04) 
Proton 3.7-5beta
Nvidia Geforce 980ti 
396.54
Intel i5-3570k

Game will not launch
Logs:
[steam-55150.log](https://github.com/ValveSoftware/Proton/files/2341835/steam-55150.log)

**Comment by LonFas** [comment]
Similarly - Game will not launch

Config:
Arch Linux
Proton 3.7-6/3.7-6-beta
MSI GAMING RX480 8G
xf86-video-amdgpu 18.1.0-1
mesa 18.2.1-1
Intel i7-3770

Logs:
[steam-proton-3.7-6.log](https://github.com/ValveSoftware/Proton/files/2412139/steam-proton-3.7-6.log)
[steam-proton-3.7-6-beta.log](https://github.com/ValveSoftware/Proton/files/2412140/steam-proton-3.7-6-beta.log)

**Comment by Zlopez** [comment]
Game not launching with `Proton 3.16-3 Beta`.

Here is proton log:
[steam-55150.log](https://github.com/ValveSoftware/Proton/files/2511921/steam-55150.log)

And here is my spec:
https://gist.github.com/Zlopez/1433022f5f75aebc3155a622997b2997

**Comment by tingvarsson** [comment]
@kisak-valve I see (https://bugs.winehq.org/show_bug.cgi?id=32515) refers to this game as well. So perhaps #753 ?
Anyway to confirm it is the same issue?

EDIT: Added [steam-55150.log](https://github.com/ValveSoftware/Proton/files/2565837/steam-55150.log)

**Comment by Makreig** [comment]
Proton 3.16-6 Beta

[steam-55150.log](https://github.com/ValveSoftware/Proton/files/2728866/steam-55150.log)

**Comment by CSahajdacny** [comment]
I have the same problem..
Ubuntu 16.04.5 LTS
core i7 4790K
2x8DDR3 2400CL11
GTX 970
[steam-55150.log](https://github.com/ValveSoftware/Proton/files/2729017/steam-55150.log)

A DRM issue, perhaps?
It will be usefull to have a list of all games with Steam CEG DRM and try them all. If none of them works, then Steam CEG is the problem here and must be solved by Valve.

**Comment by ccboxes** [comment]
I'm too.
Hardware:[https://gist.github.com/ccboxes/5f9cb0286606ebdb9a16447662119f46](Gist)
Proton: 3.16.8 beta
Log:

> 
[steam-55150.log](https://github.com/ValveSoftware/Proton/files/2996081/steam-55150.log)

**Comment by Makreig** [comment]
I have since managed to get this working by replacing the SpaceMarine.exe file.

**Comment by LonFas** [comment]
> I have since managed to get this working by replacing the SpaceMarine.exe file.

So interesting... Can you upload this file to something at cloud storages?

**Comment by kisak-valve** [comment]
Hello @LonFas, sharing game binaries on Github is not a good idea. Give https://github.com/ValveSoftware/Proton/issues/530#issuecomment-431513884 a read for a CEG workaround.

**Comment by LonFas** [comment]
> Hello @LonFas, sharing game binaries on Github is not a good idea. Give [#530 (comment)](https://github.com/ValveSoftware/Proton/issues/530#issuecomment-431513884) a read for a CEG workaround.

Thanks, i try this variant after work 😏

**Comment by Zlopez** [comment]
Doesn't start with Proton 4.2-1

Proton log: [steam-55150.log](https://github.com/ValveSoftware/Proton/files/3024035/steam-55150.log)

[System spec](https://gist.github.com/Zlopez/1433022f5f75aebc3155a622997b2997)

**Comment by Makreig** [comment]
@LonFas a simple search should be able to find you a copy.
I found it out after reading throught the reports on ProtonDB
https://www.protondb.com/app/55150

**Comment by tele1** [comment]
My log https://github.com/tele1/Steam.logs/blob/master/steam.55150.Warhammer_40000_SpaceMarine.log

**Comment by tele1** [comment]
Proton 4.2-5
Game not working, log: https://github.com/tele1/Steam.logs/blob/master/2019.05.25/Warhammer.SpaceMarine.25.05.2019.steam-55150.log.zip

**Comment by tele1** [comment]
Proton 4.2-7 game not working
Hardware Info: https://github.com/tele1/Steam.logs/blob/master/Hardware%20Info
Log: https://github.com/tele1/Steam.logs/blob/master/2019.06.12/Warhammer.Space%20Marine.Crash.2019.06.12..steam-55150.log.zip

**Comment by Weirdo1312** [comment]
> Proton 4.2-7 game not working
> Hardware Info: https://github.com/tele1/Steam.logs/blob/master/Hardware%20Info
> Log: https://github.com/tele1/Steam.logs/blob/master/2019.06.12/Warhammer.Space%20Marine.Crash.2019.06.12..steam-55150.log.zip

still does not work on latest proton (4.11-5)

**Comment by nate2014jatc** [comment]
Hello, 
I am not able to open the game running Proton-4.11-7. I will attach the log below, and keep hunting around for work-arounds in the mean time.
[steam-55150.log](https://github.com/ValveSoftware/Proton/files/3710343/steam-55150.log)

**Comment by ghost** [comment]
> I have since managed to get this working by replacing the SpaceMarine.exe file.

It works but it crashes after the 10th mission, after placing a power source in a machine. You go up an elevator and the game crashes (something that does not happen in Windows). Should I open a new issue about that? (I'm not sure how this github thing works)

**Comment by omicron-b** [comment]
@dropestein no, Proton uses 1 issue per non-supported game. Also (in my opinion) you should not open any issues about illegal versions of games, they will not be supported.

It is best to write an email to the developer and ask to support Proton. Fixing Proton issues in a Windows-only game is not something new, see No Man's Sky, Wolcen: Lords of Mayhem.

**Comment by ghost** [comment]
> @dropestein no, Proton uses 1 issue per non-supported game. Also (in my opinion) you should not open any issues about illegal versions of games, they will not be supported.
> 
> It is best to write an email to the developer and ask to support Proton. Fixing Proton issues in a Windows-only game is not something new, see No Man's Sky, Wolcen: Lords of Mayhem.

No, I mean, I have the game legally purchased on Steam. I just used that fix so I can play it in Linux too as other users here suggested. I thought that maybe that information about a crash within the game could be useful, because several people have been marking it as playable in ProtonDB because of that "fix". But, ok, I understand that at this point this is a problem from the developer then.  Thanks!

**Comment by amz-x** [comment]
Hi,

I am not able to open the game running Proton-5.0-9. 

Proton log: [steam-55150.log](https://github.com/ValveSoftware/Proton/files/4842615/steam-55150.log)

**Comment by kisak-valve** [comment]
Warhammer 40,000: Space Marine text not displaying correctly

Issue transferred from https://github.com/ValveSoftware/Proton/issues/4127.
@JordanPlayz158 posted on 2020-08-09T01:42:00:

# Compatibility Report
- Name of the game with compatibility issues: Warhammer 40,000: Space Marine
- Steam AppID of the game: 55150

## System Information
- GPU: GTX 1050 Ti 4GB (Dual Fan, Asus)
- Driver/LLVM version: 4.6.0 NVIDIA 440.100
- Kernel version: 5.7.9-1-MANJARO
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/JordanPlayz158/21255a17e2975a08694bd4ede69e8fc1
- Proton version: Proton 5.0-9

## I confirm:
- [X] that I haven't found an existing compatibility report for this game.
- [X] that I have checked whether there are updates for my system available.



## Symptoms 
The text is messed up (I believe due to corefonts being missing and if I am correct how would I install corefonts into proton manually because iirc there are reasons (licensing issues) for why proton can't include corefonts in the proton files) (See attached image for visual of messed up text)
![2020-08-08_17-35](https://user-images.githubusercontent.com/22566618/89723072-9c26f080-d9bf-11ea-9a8b-307a379ec167.png)


## Reproduction
Install Warhammer 40,000: Space Marine
Run it under Proton 5.0-9

**Comment by CSahajdacny** [comment]
Hey, @JordanPlayz158 What did you do for to make this game work?
Is a CEG infected game.. Games with CEG don't work via Proton.

**Comment by JordanPlayz158** [comment]
@CSahajdacny oh yeah I forgot to mention, I am doing what the people on ProtonDB said which is get a cracked exe of SpaceMarine.exe (before you ask, no I didn't pirate it, you can check my steam library (i didn't think anyone was gonna ask but just putting that as a precautionary measure))

**Comment by lucifertdark** [comment]
> @CSahajdacny oh yeah I forgot to mention, I am doing what the people on ProtonDB said which is get a cracked exe of SpaceMarine.exe (before you ask, no I didn't pirate it, you can check my steam library (i didn't think anyone was gonna ask but just putting that as a precautionary measure))

It's an unofficial patch.

**Comment by JordanPlayz158** [comment]
Sadly it's unofficial but it's the best thing we got so far but as I asked, does anyone know if you can install corefonts into proton to fix the messed up text?

**Comment by Doomsdayrs** [comment]
> It's an unofficial patch.
From where?

**Comment by at46** [comment]
I tried the Anniversary Edition Update today but the game does not start for me. Had hoped they had removed CEG with the update but it seems it is still there.

**Comment by dennis1248** [comment]
I can confirm that the Anniversay Edition makes no difference, game does not launch. Time to make a ruckus about it on the Space Marine Steam game forums and beg for DRM removal..

**Comment by massatt212** [comment]
Come on Valve i want to play Hitman Absolution, Grid2 and Warhammer 40K Space Marines its your DRM, it should be an EASY FIX since its your DRM come on look into this please.

**Comment by d10sfan** [comment]
[steam-55150.log](https://github.com/ValveSoftware/Proton/files/7438633/steam-55150.log)

Seeing a black screen and then a crash with the latest steam client beta and experimental.

**Comment by Raklodder** [comment]
![0001](https://user-images.githubusercontent.com/63820902/139477343-c0f04bd5-5396-4458-a930-71f76d98d982.png)
![0002](https://user-images.githubusercontent.com/63820902/139477359-9859c679-9343-47f8-b10f-de3bd7aa7434.png)


- RX 580
- MESA 21.1.8 / LLVM 12.0.1
- Kernel 5.14.14
- Fedora 34 (gnome/mutter 40.5 on wayland)
- Full system information: https://gist.github.com/3a71bbae0caac6ffe9a8d1ff9b51ac1f

I tried running the game with the latest CEG update, but when I hit play a small install window comes up (which lets me know the installation is about to complete; though it already was?), then a small black window similar to a terminal pops up with a blinking underscore, they then both disappear and the game keeps loading for another 20-30 seconds in the background (still looking at my steam game library) before the game attempts to launch and crash during the initial loading blackscreen.

1635348752 experimental-6.3-20211027
Proton game log (crash): https://gist.github.com/06596c160a615872ebce4e841b85508d

1635525635 experimental-bleeding-edge-6.3-1821-20211029-paf96ce-w78e167-d1b1206-v430c77
Proton game log (crash): https://gist.github.com/a98c7d435ef086fd77f7b083da7b8721

![0003](https://user-images.githubusercontent.com/63820902/139594946-5129dca0-2c77-4281-aba6-d710d7cd9b42.jpg)

# 20211030 update:
I removed Proton 6.3-7 from my system, restart steam and installed the latest bleeding-edge update and now the game runs!
- Audio and video playback works. Mouse and keyboard input works.

_1635544687 experimental-bleeding-edge-6.3-1836-20211029-paf96ce-w8a5bac-d1b1206-v430c77_

# 20211030 update 2:
I can play for about 30 sec before the game begin to lag/stutter, then goes back to normal and then lag/stutter some more.

Also my logfiles are huge and I'm having difficulties opening them, where the same few lines are repeated over and over again.

```
7998.190:010c:0198:trace:seh:__regs_MSVCRT__setjmp3 buf=27A4FE10 ebx=00000000 esi=106ae154 edi=00000032 ebp=27a4fe5c esp=27a4fdf8 eip=0074e6cc frame=27a4fef8
7998.190:010c:0198:trace:seh:MSVCRT_longjmp buf=27A4FE10 ebx=00000000 esi=106ae154 edi=00000032 ebp=27a4fe5c esp=27a4fdf8 eip=0074e6cc frame=27a4fef8 retval=00000001
7998.190:010c:0198:trace:seh:MSVCRT_longjmp cur_frame=27a4fef8
7998.197:010c:0198:trace:seh:__regs_MSVCRT__setjmp3 buf=27A4FD94 ebx=00000000 esi=106ae154 edi=00000032 ebp=27a4fde0 esp=27a4fd7c eip=0074e6cc frame=27a4fe64
7998.197:010c:0198:trace:seh:MSVCRT_longjmp buf=27A4FD94 ebx=00000000 esi=106ae154 edi=00000032 ebp=27a4fde0 esp=27a4fd7c eip=0074e6cc frame=27a4fe64 retval=00000001
7998.197:010c:0198:trace:seh:MSVCRT_longjmp cur_frame=27a4fe64
```
https://gist.github.com/899d7caee251fc710f2f198bba096522

# 20211031 update 3:
I'd forgot to add gamemoderun %command% in my game launch options and now it runs great (as it no longer falls back to 10-25% CPU utilization). However though it may not be game specific, I'm experiencing some strange behavior when launching other non-CEG games. Not only do they take longer to launch, but the new black window keeps popping up. This also happens each time I log into Steam, where my CPU usage goes through the roof, black window appears and then nothing.

**Comment by PyDario** [comment]
I've had a similiar experience as @Raklodder when I changed to the bleeding-edge version of Proton Experimental and Steam Beta
Specs:

- GPU: GTX 1070
- CPU: i5 6600K
- RAM: 16 GB DDR4
- Drivers: nvidia 470.74
- Kernel: 5.14.12-zen1-1-zen 
- OS: Arch with KDE Plasma

I've got the game to run, however it ran at much worse performance than with a drm-free .exe

- On startup, the game doesn't show, only Audio is playing until the AMD logo comes up
- Framerate is lower in areas where nothing is happening
- On an enemy encounter, the framerate dropped significantly
- At the same time, the audio began to crack and break at some points during the fight
- Unlike Raklodder, the performance did not recover after a while

**Comment by turtlewit** [comment]
Following the update to Proton Experimental adding initial support for Steam’s CEG DRM, this game now works flawlessly with no tweaks. As reported above, Steam Beta must be enabled and the game must be run on Proton Experimental Bleeding Edge. Performance for me was perfect, no stutters or hitches. KBM and Gamepad input worked. Did not test multiplayer.

Specs:
* Distro: Arch Linux + Wayland (Sway)
* Kernel: 5.14.14-zen1-1-zen
* RAM: 24 GB
* GPU Driver: 4.6 Mesa 21.2.4
* GPU: AMD Radeon RX Vega 56 (VEGA10, DRM 3.42.0, 5.14.14-zen1-1-zen, LLVM 12.0.1)
* CPU: AMD Ryzen 9 3900X 12-Core

**Comment by dennis1248** [comment]
Whatever was bothering the game previously seems to be fixed in the latest Proton Experimental bleeding edge. It now runs perfectly and without performance issues.

- Arch Linux
- 5.14.14-zen
- Mesa 21.2.4
- AMD RX 5700
- Proton Experimental (bleeding edge 01-11-2021)

**Comment by PyDario** [comment]
I can confirm that with the latest update, the issues I previously reported have all been remedied.

**Comment by Zlopez** [comment]
I'm having issues with mouse not working in menu, otherwise the games runs smoothly. I even played one MP match.

I'm running Proton Experimental with latest update.

Specs: https://gist.github.com/Zlopez/1433022f5f75aebc3155a622997b2997

**Comment by d10sfan** [comment]
With the latest experimental, after I re-installed the game  and removed the prefix, and now it seems to be working

**Comment by Devorlon** [comment]
I can confirm that Space Marine is working:

- Linux 5.14.16-arch1-1
- 21.2.4-1
- RX 6800XT
- Proton Experimental (as of writing)

**Comment by Devorlon** [comment]
@kisak-valve I think this can be closed.

**Comment by adupuis** [comment]
In my case, the game is showing a black screen and crashes immediately. Proton experimental or 6.3-8.

**Comment by tele1** [comment]
**adupuis**  <details><summary>Check This</summary><p>
# Compatibility Report
- Name of the game with compatibility issues:
- Steam AppID of the game:

## System Information 
( I like use inxi tool, example command:  **inxi -bz** )
- GPU: 
- Driver/LLVM version: 
- Kernel version: 
- Link to full system information report as [Gist](https://gist.github.com/):
- Proton version:

## I confirm:
- [ ] that I haven't found an existing compatibility report for this game.
- [ ] that I have checked whether there are updates for my system available.



## Symptoms 


## Reproduction

1. You can find the Steam AppID in the URL of the shop page of the game.
   e.g. for `The Witcher 3: Wild Hunt` the AppID is `292030`.
2. You can find your driver and Linux version, as well as your graphics
   processor's name in the system information report of Steam.
3. You can retrieve a full system information report by clicking
   `Help` > `System Information` in the Steam client on your machine.
4. Please copy it to your clipboard by pressing `Ctrl+A` and then `Ctrl+C`.
   Then paste it in a [Gist](https://gist.github.com/) and post the link in
   this issue.
5. Please search for open issues and pull requests by the name of the game and
   find out whether they are relevant and should be referenced above.

</p></details>


=================================================
However for me also worked / working ( I hope ) very well with Proton experimental. ( I have no objections  )
OS:   Linux Mint 20.2 Uma
GPU:    NVIDIA GeForce GTX 660
driver: nvidia v: 470.82.00

**Comment by saschalippert** [comment]
got the same issue as adupuis. the game starts, a small back screen shows up and then crashes to the bug report window.

proton:
experimental, experimental bleeding edge and 6.3.8

inxi:
https://gist.github.com/saschalippert/e574c809dd8c60f44cce251b157537cd

steam sys info:
https://gist.github.com/saschalippert/3388f7f584d0cbe8dd3f5c5f089a5aac

space marine steam logs (experimental):
https://gist.github.com/saschalippert/47114de135e1e050181a5a68c3d98cc1

**Comment by ChaosBlades** [comment]
For those that can't get this to launch without crashing you can't have more than 24 threads. It crashes somewhere between 25 and 28 threads. We discovered the same issue with Far Cry 4 where it won't launch with somewhere between 29 and 32 threads.

https://github.com/ValveSoftware/Proton/issues/2143

**Comment by ChaosBlades** [comment]
If you have more than 26 threads on your CPU you will need to use this launch option or the game will crash as soon as you go full screen.

`WINE_CPU_TOPOLOGY=26:0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25 %command%`

**Comment by GloriousEggroll** [comment]
> If you have more than 26 threads on your CPU you will need to use this launch option or the game will crash as soon as you go full screen.
> 
> `WINE_CPU_TOPOLOGY=26:0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25 %command%`

Lara Croft and the Guardian of Light https://github.com/ValveSoftware/Proton/issues/249 has the same problem. Failed with this value but worked with 12 cores specified 

```
WINE_CPU_TOPOLOGY=12:0,1,2,3,4,5,6,7,8,9,10,11 %command%
```

**Comment by makeclean** [comment]
Now interestingly, I still have the same issue as others, black screen, bugsplat, the WINE_CPU_TOPOLOGY works great for FC4, but not helping here it seems. Please see the attached log.

[steam-55150.log](https://github.com/ValveSoftware/Proton/files/9066437/steam-55150.log)

**Comment by tele1** [comment]
**makeclean**  try write something about your hardware.

<details><summary>Tip how give info ( click this to expand info )</summary><p>
# Compatibility Report
- Name of the game with compatibility issues:
- Steam AppID of the game:

## System Information 
( I like use inxi tool, example command:  **inxi -bz** )
- GPU: 
- Driver/LLVM version: 
- Kernel version: 
- Link to full system information report as [Gist](https://gist.github.com/):
- Proton version:

## I confirm:
- [ ] that I haven't found an existing compatibility report for this game.
- [ ] that I have checked whether there are updates for my system available.



## Symptoms 


## Reproduction

1. You can find the Steam AppID in the URL of the shop page of the game.
   e.g. for `The Witcher 3: Wild Hunt` the AppID is `292030`.
2. You can find your driver and Linux version, as well as your graphics
   processor's name in the system information report of Steam.
3. You can retrieve a full system information report by clicking
   `Help` > `System Information` in the Steam client on your machine.
4. Please copy it to your clipboard by pressing `Ctrl+A` and then `Ctrl+C`.
   Then paste it in a [Gist](https://gist.github.com/) and post the link in
   this issue.
5. Please search for open issues and pull requests by the name of the game and
   find out whether they are relevant and should be referenced above.

</p></details>

**Comment by ahjolinna** [comment]
update...
apparently the issue was that two of the game files was broken/missing where still broken even thought I had already checked and redownloaded them when I made this test, but now I when retested it again (as I notice the "wednesday" update), it found those two files where yet again broken/missing.... (not sure why, I'm sure it did download them last time...oh well)

...but at least NOW the game works ...so yay

--------

I just tried on SteamDeck and it fails to launch, I get the same black screen & bugsplat issue.

I'm using SteamOS beta(-main) and tried with both 7.0.4 and experimental bleeding edge
here is the log for the experimental : [steam-55150.log](https://github.com/ValveSoftware/Proton/files/9924309/steam-55150.log)

**Comment by gogoprog** [comment]
Same issue here, and as @ahjolinna encountered, somehow 2 files were missing.
After having verified the integrity of the local files, they were downloaded again and now the game starts.

**Comment by baufaus** [comment]
I have pretty much the same issue, on an RTX 2080 with the newest drivers, and i9 10850k (20 threads). Nothing works (tried all possible fixes here), the farthest I can get is bugsplat crash window by use of <PROTON_USE_WINED3D11=1 %command%>. If anyone has figured this out and forgotten to post their fix, that would be much appreciated.

- Arch Linux, latest
- KDE Plasma, latest 
- RTX 2080, latest
- i9 10850k, 10 cores 20 threads
- Tried on ~10 proton versions, including GE

**Comment by frozen-sea** [comment]
Adding my 2 cents, since this is still open. Installed game and tried running with current Proton Experimental bleeding-edge. First attempt froze on logo screen, killed the process to add PROTON_LOG and try again, the game then worked flawlessly. It's pretty spammy when working, so adding the log as reference.

* Nobara 38, kernel 6.4.8
* KDE Plasma Wayland
* RTX 4070 Ti, driver 535.98
* Ryzen 5800X3D (16 threads)

[steam-55150.zip](https://github.com/ValveSoftware/Proton/files/12353001/steam-55150.zip)

**Comment by ahjolinna** [comment]
I just checked and it works just fine on my Desktop, and with SteamDeck the SteamOS beta/main (Build: 20230807.1000)  it does open just fine to the menu **BUT** the controls won't work any more ...so that's my (new) issue.
at least it seems the old "bug-splash/ 2 files missing" issue is gone now


_**both are tested using proton experimental bleeding edge.**_

my desktop spec:

```
- Operating System: openSUSE MicroOS 20230814
- KDE Plasma Version: 5.27.7
- KDE Frameworks Version: 5.108.0
- Qt Version: 5.15.10
- Kernel Version: 6.4.9-1-default (64-bit)
- Graphics Platform: Wayland
- Processors: 16 × AMD Ryzen 7 5700G with Radeon Graphics
- Memory: 46.8 GiB of RAM
- Graphics Processor: NVIDIA GeForce RTX 3060 Ti/PCIe/SSE2 (Driver: v535.86.05)
```

**Comment by frozen-sea** [comment]
Can confirm, controller is not recognized. Also tested Proton-GE and it has the same issue. I tried protontricks xinput, but the only thing I managed to achieve was crashing the game.

**Comment by ahjolinna** [comment]
I just also tested on steamdeck with proton 8.0 and I had both issues, the old "bugsplash/2files missing" and now the **new** controller is not recognized

**Comment by ahjolinna** [comment]
@kisak-valve there seems to be a controller regression at least for this game

**Comment by kisak-valve** [comment]
Hello @ahjolinna, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options, reproduce the regression, and attach the generated $HOME/steam-$APPID.log to this issue report as a file. (Proton logs compress well if needed.) Also, can you verify what the current behavior is with Proton 7.0 and this game?

**Comment by ahjolinna** [comment]
@kisak-valve I've tested proton 7.0 has now also this controller regression, I have the log file: 
[steam-55150 (1).log](https://github.com/ValveSoftware/Proton/files/12371529/steam-55150.1.log)

and as bonus if needed:
here is the log file for experimental (bleedingedge)...when the old "bugsplash/2files missing" (still seems to happen):[steam-55150 (2).log](https://github.com/ValveSoftware/Proton/files/12371624/steam-55150.2.log)

and here is the experimental log file after when controls dont work: [steam-55150 (3).log](https://github.com/ValveSoftware/Proton/files/12371653/steam-55150.3.log)

PS controls didnt work on the menu but they did work to skipping the all the logos before the menu, and the touchscreen worked (as a mouse input)

**Comment by alasky17** [comment]
@ahjolinna It sounds like this is a regression in SteamOS Beta/Main rather than Proton.  Could you please try SteamOS stable and see if the controller regression is present there?

**Comment by ahjolinna** [comment]
@alasky17 issues still exist on SteamOS stable with proton-experimental (**none** bleeding edge version), here is log: [steam-55150 (5).log](https://github.com/ValveSoftware/Proton/files/12422498/steam-55150.5.log) and here is log for proton-8.0-3c : [steam-55150 (6).log](https://github.com/ValveSoftware/Proton/files/12422558/steam-55150.6.log)

**Comment by alasky17** [comment]
@ahjolinna Was there ever a time/location where the controller was working?

**Comment by ahjolinna** [comment]
@Zlopez worked fine at least on November 2022 https://github.com/ValveSoftware/Proton/issues/934#issuecomment-1301336336 ...just had the same old "bugsplash/2 files missing issue" (not an issue on desktop)

the weird part is that control works to skip all the logos and the "press any key" section before the menu, but in the menu nothing works (just touch screen)

**Comment by AJuujarvi** [comment]
@ahjolinna I've tried looking into this bug and I cannot reproduce your issue no matter what I do. Can you make sure your Deck controller template for the game is set to "Gamepad with Joystick Trackpad"? Additionally, can you "Verify Integrity of Game Files"? This is found by clicking the Gear to the right of the Play button, clicking Properties, scrolling down to Installed Files, and there being a button for it on the right.

As another option, we can try deleting the prefix. First we should back up your current one. That should be found in ~/.steam/steam/steamapps/compatdata as the directory "55150". Create a backup of that one. Then delete the original. Then try rebooting the game, and see if recreating the prefix will fix the issue. The reason we create a backup is that there are game files like saves that are in there occasionally you would want to potentially bring over.

Let me know if any of that helps. If it still does not, can you create another log please and attach it, this time with the launch command `PROTON_LOG=+hid,+hidp,+hid_report,+plugplay,+rawinput,+dinput,+xinput,+joystick,+setupapi,+input %command%`

**Comment by ahjolinna** [comment]
@AJuujarvi hmm weird, I was using the Recommended controller Template  / the Official Layout for game ..which is the "Gamepad with Joystick Trackpad" ...I tried to re-apply it ...no difference. Also I HAVE to do the "Verify Integrity of Game Files" thing for the game to launch when get the "black screen & bugsplat issue" ...which I always get on fresh install, and for some reason it always finds the 2 files, and after that it fixes THAT old issue.

anyway, I uninstall and reinstall the game and checked that "55150" was removed, and tried everything again:
, 
here is log file with proton 8.0 with bugsplat thing: [steam-55150 (7).log](https://github.com/ValveSoftware/Proton/files/12439199/steam-55150.7.log)


**and here is the log file with the control issue**, which is 248.8MB:  https://www.swisstransfer.com/d/f4041cae-c588-454b-8256-35de45d8ec66 (link valid for 30days/250 downloads)




(both have the launch command you provided)

**Comment by jefferyrlc** [comment]
Cleared the prefix and tried launching with Proton-8.0-4 and it's crashing on launch - log attached
[steam-55150.log](https://github.com/ValveSoftware/Proton/files/13748729/steam-55150.log)

**Comment by mathew2214** [comment]
game doesnt launch
Debian 12 Bookworm
here's my sysinfo
https://gist.github.com/mathew2214/715578ce13a299707754929625d62d71
https://gist.github.com/mathew2214/92756432d6b249748d1280db6bc2820c

**Comment by kneo** [comment]
OS: Debian 12 Bookworm using this [installation method](https://wiki.debian.org/Steam)

Starting it the first time I got a steam log (1.). But now when trying to start the game again does not even create the steam log file anymore.
It just hangs with "Launching, Updating Executable" and nothing more happens. After a while the grayed out button saying "launching" becomes green and says "Play" again.

It spits out a dmp (2.) file and the following lines in the log:

```
[2024-09-22 14:41:46] Nothing to do
src/clientdll/clientjobrequestcustombinary.cpp (605) : eResult != k_EResultRemoteCallFailed
src/clientdll/clientjobrequestcustombinary.cpp (605) : eResult != k_EResultRemoteCallFailed
assert_20240922144151_55.dmp[32188]: Uploading dump (out-of-process)
/tmp/dumps/assert_20240922144151_55.dmp
assert_20240922144151_55.dmp[32188]: Finished uploading minidump (out-of-process): success = yes
assert_20240922144151_55.dmp[32188]: response: CrashID=bp-b3722176-5010-45e8-9aa2-1f2832240922
assert_20240922144151_55.dmp[32188]: file ''/tmp/dumps/assert_20240922144151_55.dmp'', upload yes: ''CrashID=bp-b3722176-5010-45e8-9aa2-1f2832240922''
```

The game however does not start at all. It did once initially but crashed at the bug splat screen. But that never happened again after it happened only once. Tried reinstalling the game, and steam but the problem persist.

1.) [steam-55150-#1.log](https://github.com/user-attachments/files/17089779/steam-55150-.1.log)

2.) [assert_20240922144151_55.dmp.zip](https://github.com/user-attachments/files/17089778/assert_20240922144151_55.dmp.zip)

**Comment by mathew2214** [comment]
Update: I was able to launch it just fine by reverting to Proton 7.

**Comment by kneo** [comment]
I tried several proton versions too, also setting the core topology, but it does not even launch. Same behavior. I suspect something in steam is stuck, rather than proton, because I could work with the logs most of the time to get things to work. But here is no helpful information, neither online nor in logs. Rather unfortunate, since the new SM2 just worked! :(

**Comment by kneo** [comment]
After endless trying I got the Game to "just work" as well :)

Works on Proton 9.0-3, possibly any other version, without any launch options.

But, I needed to start it on Windows initially. With that done the first time set up under Linux did finally work, and did no longer crash, or what ever my observed behavior is called.

Next stop the "bug splat" screen.

Tried the Play on Linux guide to get it to run but it did not work, kept the with protontricks installed components installed though.

It seems to be a DRM issue, as replacing the SpaceMarine.exe (Version 1.0.165) with an _unofficial fixed one_, did the trick.

For obvious reasons I can not share the source of that file; because the emperor does not approve such heresy ; but thanks to *those guys* I can finally play my game :)

**Comment by makeclean** [comment]
So I'm back, this used to work on Fedora 39 (I think) but since upgrading to Fedora 40 and 41, this no longer works. I suspect that same thing as @kneo a DRM issue (maybe at the kernel level?). I would love to know where @kneo got the patched version :)

**Comment by kneo** [comment]
Just send me a Mail from an "unprofessional" provider I'll send the details. No need to get into trouble from your employer over this. ^^

**Comment by mathew2214** [comment]
debian 12, GE-Proton7-35, navi10,  game launches fine when using PROTON_USE_WINED3D11=1, but audio underruns like crazy.

**Comment by MRDGH2821** [comment]
I got this game from Amazon Prime & Installed it via Heroic launcher.
Launching the game via Proton 10.0 beta gives 27 FPS. 

But using GE-Proton 9-27 and 8-32 (after deleting prefixes), gave 70+ FPS.

Rest of the options were at their defaults.
